GNUmakefile

```diff
@@ -3,11 +3,12 @@
 ############################################################################
 
 # Set this to change the target installation path
-PREFIX   = /usr/local
-BINDIR   = ${PREFIX}/bin
-SHAREDIR = ${PREFIX}/share
-MANDIR   = ${SHAREDIR}/man
-DISTDIR ?= dist
+PREFIX     = /usr/local
+BINDIR     = ${PREFIX}/bin
+SYSCONFDIR = ${PREFIX}/etc/direnv
+SHAREDIR   = ${PREFIX}/share
+MANDIR     = ${SHAREDIR}/man
+DISTDIR   ?= dist
```

Comment:
In this changeset, 6 lines are modified while the only relevant line to review is the 3rd one.

Reply:
@drupol done